ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import logging
+from asyncio import run
+from json import loads
+from typing import TYPE_CHECKING
+
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import NoAliasDumper
+from ansible_collections.arista.avd.roles.eos_validate_state.python_modules.constants import ACRONYM_CATEGORIES, AVD_TEST_CLASSES
+
+from .ansible_eos_device import AnsibleEOSDevice
```

Comment:
We probably want this to come from the action plugin file as well as an argument, since a lot of Ansible artifacts are imported in `ansible_eos_device`. It can wait for next phase, so just add some TODO.

Reply:
Addressed in [c6d9d58](https://github.com/aristanetworks/ansible-avd/pull/3171/commits/c6d9d585cae5d24e71ea981859c3321165a4d515)